rfc6962-bis.xml

```diff
@@ -612,6 +612,9 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
         <t>
           <spanx style="verb">sth_extensions</spanx> matches the STH extensions of the corresponding STH.
         </t>
+        <t>
+          The length of the NodeHash MUST match HASH_SIZE of the log.
```

Comment:
"the NodeHash" doesn't sound quite right, because "the" implies it's singular.

I suggest dropping the "the".  So just "The length of NodeHash..."